web/src/locales/fr/data.json

```diff
@@ -244,12 +248,14 @@
     "Please input your password, at least 6 characters!": "Veuillez entrer votre mot de passe, au moins 6 caractères !",
     "Please input your username, Email or phone!": "Veuillez entrer votre nom d'utilisateur, votre e-mail ou votre téléphone!",
     "Sign In": "Se connecter",
+    "Sign In With WebAuthn": "Sign In With WebAuthn",
     "Sign in with code": "Se connecter avec le code",
     "Sign in with password": "Se connecter avec le mot de passe",
     "Sign in with {type}": "Se connecter avec {type}",
     "Signing in...": "Connexion en cours...",
     "The input is not valid Email or Phone!": "L'entrée n'est pas valide Email ou Téléphone !",
     "To access": "Pour accéder à",
+    "WebAuthn": "WebAuthn",
```

Comment:
`WebAuthn` doesn't need translation anywhere. I don't think you need to wrap it with i18n.